plugin/mlsdbonlinelocator.cpp

```diff
@@ -45,8 +46,33 @@ MlsdbOnlineLocator::MlsdbOnlineLocator(QObject *parent)
     , m_modemManager(new QOfonoExtModemManager(this))
     , m_simManager(0)
     , m_networkManager(new NetworkManager(this))
+    , m_fallbacksLacf(true)
+    , m_fallbacksIpf(true)
     , m_currentReply(0)
 {
+    QFile gpsConf(QStringLiteral("/system/etc/gps_sfos.conf"));
+    if (gpsConf.open(QIODevice::ReadOnly)) {
+
+        while (!gpsConf.atEnd()) {
+            const QByteArray line = gpsConf.readLine().trimmed();
+            if (line.startsWith('#'))
+                continue;
+
+            const QList<QByteArray> split = line.split('=');
+            if (split.length() != 2)
+                continue;
+
+            const QByteArray key = split.at(0).trimmed();
+            const QByteArray data = split.at(1).trimmed();
+
+            if (key == "MLS_FALLBACKS_LACF" && data == "FALSE") m_fallbacksLacf = false;
+            if (key == "MLS_FALLBACKS_IPF" && data == "FALSE") m_fallbacksIpf = false;
+        }
+    }
```

Comment:
This looks like something that `QSettings` could do for you. e.g. `QSettings s(fileName, QSettings::IniFormat)`.